### ISSUE TITLE
rest_wsgi broken under mod_wsgi

### ISSUE THREAD

**Issue by czee** [vmware-archive/salt-api#107]
Below is the output from /var/log/apache2/error.log. The salt-api script is broken when you make a call to Apache running mod_wsgi:

```
[Wed Aug 07 10:05:17 2013] [error] [client 192.168.100.8]   File "/usr/lib/pymodules/python2.7/saltapi/netapi/rest_wsgi.py", line 246, in saltenviron
[Wed Aug 07 10:05:17 2013] [error] [client 192.168.100.8]     environ['SALT_OPTS'] = __opts__
[Wed Aug 07 10:05:17 2013] [error] [client 192.168.100.8] NameError: global name '__opts__' is not defined
```

**Comment by czee** [comment]
Closed by whiteinge in 25304c9b1e6cedeec3bf3ebd87d04c06956e23e8

**Comment by whiteinge** [comment]
Thanks for filing this, then updating it. :)  Looks like we barely missed each other on IRC.
